### PR TITLE
Add happy path spec for Text Input widget

### PR DESCRIPTION
**Pull request by ajith-k-v**
Fixes: #4014 

Screenshot 2022-09-14 at 12 33 29 PM

### CODE REVIEW

File: cypress-tests/cypress/constants/texts/textInput.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export const textInputText = {
+  customValidation: (name, message,) => {
+    return ["{{",`components.${name}.value ? true : '${message}'}}`,];
+  },
+
+  defaultWidgetName: "textinput1",
+  textInputRegex: "^[A-Z]*$",
+  textMinimumLength: "4",
```
Comment (emidhun) [inline cypress-tests/cypress/constants/texts/textInput.js:8]: Can we add some limited random value for max and min length?
eg: [1-10]

File: cypress-tests/cypress/e2e/editor/widget/textInputHappyPath.cy.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    openAccordion(commonWidgetText.accordionProperties);
+    verifyAndModifyParameter(commonWidgetText.labelPlaceHolder, data.customText);
+    cy.forceClickOnCanvas();
+    cy.get(commonWidgetSelector.draggableWidget(data.widgetName))
+    .invoke('attr', 'placeholder').
+    should('contain', data.customText)
+
+    openEditorSidebar(data.widgetName);
+    openAccordion(commonWidgetText.accordionEvents);
+    addDefaultEventHandler(data.alertMessage);
```
Comment (emidhun) [inline cypress-tests/cypress/e2e/editor/widget/textInputHappyPath.cy.js:63]: I think for widgets with exposed values, We can use the alert message with the value.
Please share your thoughts.

**Review by emidhun — COMMENTED** [review]
@ajith-k-v  LGTM,
I have a few suggestions Can we use the exposed value on alert message, csa, custom messages, etc?

**Comment by shubh22** [comment]
Task linked: [CU-2ttqq9v Add HappyPath spec for Text Input](https://app.clickup.com/t/2ttqq9v)

File: cypress-tests/cypress/constants/texts/textInput.js (hunk 1 above)
Comment (ajith-k-v) [reply cypress-tests/cypress/constants/texts/textInput.js]: Yeah, sure we can add random values for min and max length. Updated on the latest commit

File: cypress-tests/cypress/e2e/editor/widget/textInputHappyPath.cy.js (hunk 2 above)
Comment (ajith-k-v) [reply cypress-tests/cypress/e2e/editor/widget/textInputHappyPath.cy.js]: Yeah, we can use exposed values on alert messages, CSA and custom messages. Updated on the latest commit

**Comment by ajith-k-v** [comment]
@emidhun Added the review changes. Please review
Screenshot 2022-09-22 at 9 47 11 PM

**Review by emidhun — APPROVED** [review]
@ajith-k-v Looking good to me.

**Pull request merged**